### ISSUE TITLE
Onion Rings Windurst Quest

### ISSUE THREAD

**Issue by neonracer** [HomepointXI/Issues#232]
**Date & Time**: Friday October 5th, 2018 @ precisley 12 pm

**Client Version (use `/ver` in game):** ver 30180904_0

**Character Name:**  Neonracer

**Discord Name (if different from in game):** Neonracer

**Nation:**  Bastok

**Job(level)/Sub Job(level):** 80 Samurai (quest related not job related)

**NPC or Monster or item Name:**  Kohlo-Lakolo - Port Windurst G-5 

**Zone name:** Port Windy

**Coordinates (use `!where`):** X-25, Y-6, Z 189 (146)

**Item tool link (for items issues only):** Bouncer Club?

**Multi-boxing? (multiple clients on same connection):** No

**Steps to reproduce:** Not sure, I did check everything Autkast and Epictaru ( i know you two nag me alot!!)

**Any additional information:** I tired to reporoduce, but don't really care abot item, wanted to report anyways that I didt get item. Going to see if I can continue on. (got the music though) so might be able to move on this quest!

**Comment by EpicTaru** [comment]
Um, what EXACTLY is the issue in this ticket? Please clarify, @neonracer

**Comment by neonracer** [comment]
Didnt get bouncer ckub and not moving to nxt quest

**Comment by EpicTaru** [comment]
Do you have a screenshot of the point in the quest  where you're supposed to get the item?

**Comment by nfaletra** [comment]
`If you didn't get your Bouncer club please zone out and zone back in and talk to Kohlo-Lakolo once more.`

Have you tried this?
Also, what is your Fame level in Windurst? Talk to fame checker npc and let us know what dialogue they use.

Windurst Waters(North) | Zabirego-Hajigo | (F-10) In Timbre Timbers

**Comment by EpicTaru** [comment]
![image](https://user-images.githubusercontent.com/26195580/46555480-14469b80-c8a9-11e8-84e5-f918bd1308bd.png)

![image](https://user-images.githubusercontent.com/26195580/46555528-3f30ef80-c8a9-11e8-941f-62ee2af53e16.png)

Translation: You completed that quest.  Try advancing to the next quest in the line.

**Comment by neonracer** [comment]
I can..i dont thinknits a wait 24 hr quest. Its rediculously easy.

**Comment by neonracer** [comment]
Ill take a look in a few guys.

**Comment by neonracer** [comment]
![neonracer_2018 10 06_134932](https://user-images.githubusercontent.com/28202863/46574299-f93e5f00-c96e-11e8-879b-7616c2294d22.png)

Upload a pic. Kohlo Lakol just keeps saying this.

Lord NEONRACER.. That is my fame, guys!

![neonracer_2018 10 06_135521](https://user-images.githubusercontent.com/28202863/46574332-8681b380-c96f-11e8-9966-d0721d9596dc.png)

So far, that is everything, I zoned out went back in, Zilch. Checked fame guy.. I'm known to the Windy, as Lord... so.. </shrugs?>
